### PR TITLE
added okyc aadhaar verify Api's

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**

### CODE REVIEW

File: src/src/modules/auth/auth.controller.ts [hunk 1]
```diff
@@ -174,4 +174,32 @@ export class AuthController {
 	) {
 		return this.authService.verifyAadhaarKyc({ ...req, id: id }, response);
 	}
+
+	@Post('api/okyc2/aadhaar/verify')
+	@UseGuards(new AuthGuard())
+	private async okyc2AadhaarVerification(
```
Comment (manojLondhe) [inline src/src/modules/auth/auth.controller.ts:180]: rename okyc2AadhaarVerification => okyc2AadhaarVerify

File: src/src/modules/auth/auth.controller.ts [hunk 2]
```diff
@@ -174,4 +174,32 @@ export class AuthController {
...
+		return this.authService.okyc2AadhaarVerification(
+			body,
+			request,
+			response,
+		);
+	}
+
+	@Get('api/okyc2/aadhaar/verify/:id')
+	@UseGuards(new AuthGuard())
+	private async getOkyc2AadhaarVerification(
```
Comment (manojLondhe) [inline src/src/modules/auth/auth.controller.ts:194]: rename getOkyc2AadhaarVerification => getOkyc2AadhaarVerificationStatus

File: src/src/modules/auth/auth.service.ts [hunk 3]
```diff
@@ -435,7 +435,21 @@ export class AuthService {
...
 	}
+	public async okyc2AadhaarVerification(body, request, response) {
+		return await this.aadhaarKycService.okyc2AadhaarVerification(
+			body,
+			request,
+			response,
+		);
+	}
 
+	public async getOkyc2AadhaarVerification(id, request, response) {
```
Comment (manojLondhe) [inline src/src/modules/auth/auth.service.ts:446]: rename getOkyc2AadhaarVerification => getOkyc2AadhaarVerificationStatus

File: src/src/modules/auth/auth.service.ts [hunk 4]
```diff
@@ -435,7 +435,21 @@ export class AuthService {
 			response,
 		);
 	}
+	public async okyc2AadhaarVerification(body, request, response) {
```
Comment (manojLondhe) [inline src/src/modules/auth/auth.service.ts:438]: rename okyc2AadhaarVerification => okyc2AadhaarVerify

File: src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts [hunk 5]
```diff
@@ -175,6 +175,65 @@ export class AadhaarKycService {
...
+					: message,
+				code: response?.data?.error?.code
+					? response?.data?.error?.code
+					: 500,
+
+				data: {},
+			});
+		}
+	}
+	public async okyc2AadhaarVerification(body, req, resp) {
```
Comment (manojLondhe) [inline src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts:191]: add blank line at the start of new method

Comment (manojLondhe) [inline src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts:191]: rename okyc2AadhaarVerification => okyc2AadhaarVerify

File: src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts [hunk 6]
```diff
@@ -175,6 +175,65 @@ export class AadhaarKycService {
...
+
+				data: {
+					code: response?.data?.error?.code
+						? response?.data?.error?.code
+						: 500,
+				},
+			});
+		}
+	}
+	public async getOkyc2AadhaarVerification(id, req, resp) {
```
Comment (manojLondhe) [inline src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts:221]: rename getOkyc2AadhaarVerification => getOkyc2AadhaarVerificationStatus

Comment (manojLondhe) [inline src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts:221]: add blank line at the start of new method

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/src/modules/auth/auth.controller.ts (hunk 2 above)
Comment (yogesh-pawar-2023) [reply src/src/modules/auth/auth.controller.ts]: added as per comment

File: src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts (hunk 6 above)
Comment (yogesh-pawar-2023) [reply src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts]: added as per comment

**Pull request merged**